.github/workflows/auto-changelog.yml

```
@@ -0,0 +1,100 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+
+      - name: Install conventional-changelog-cli
+        run: npm install -g conventional-changelog-cli
+
+      - name: Generate changelog
+        run: |
+          # Generate changelog using .versionrc config
+          conventional-changelog -p angular -i CHANGELOG.md -s --config .versionrc
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)